.github/components.yml

```diff
@@ -70,11 +68,6 @@ TEMPLATE:
 
 AUTO:
   revalidate:
-    - CPU
-    - GPU
-    - GNA
-    - HETERO
-    - AUTO_BATCH
     - TEMPLATE
```

Comment:
do we need to keep `TEMPLATE` here?

Reply:
yes,  currently I found device as AUTO:TEMPLATE instantiated in template as well